modules/create_timepoint/.gitignore

```diff
@@ -0,0 +1 @@
+js/*
```

Comment:
js/* means all js file should be ignored. 
How about this modules/create_timepoint/js/createTimepointIndex.js?

Reply:
@driusan is the conventional standard removing all .js files in the js directory? Otherwise should I remove just the js file in .gitignore as @kongtiaowang suggests.

Reply:
the convention is generally "if there are some pre-React .js files already tracked there, ignore individual files, otherwise ignore all files in the js directory"